engine/installation/linux/rhel.md

```diff
@@ -27,6 +27,13 @@ version of Docker, remove it using the following command:
 $ sudo yum -y remove docker
 ```
 
+You may also have to remove the package `docker-selinux` which will conflict with
```

Comment:
I did a quick Google and it looks like this applies to CentOS, Fedora, and Oracle as well (but on Oracle, the package name is `docker-engine-selinux`), so if you could add it to the CentOS and Oracle docs as well, that would be great.